src/services/Dogs.js

```diff
@@ -0,0 +1,11 @@
+import { client, checkError } from './client';
+
+export async function fetchDogs() {
+  const resp = await client.from('dogs').select().order('name');
+  return checkError(resp);
+}
+
+export async function fetchDogId(id) {
+  const resp = await client.from('dogs').select('*').match({ id });
```

Comment:
Could also use `.single()` to get a single object instead of an array with a single object
https://supabase.com/docs/reference/javascript/single